samplePDF/samplePDFFDBase.h

```diff
@@ -134,8 +125,9 @@ class samplePDFFDBase :  public samplePDFBase
   bool IsEventSelected(const std::vector<std::string>& ParameterStr, const int iSample, const int iEvent);
   bool IsEventSelected(const std::vector<std::string>& ParameterStr, const std::vector<std::vector<double>> &SelectionCuts, const int iSample, const int iEvent);
 
+  /// @brief Check whether a normalisation systematic affects an event or not
   void CalcXsecNormsBins(int iSample);
-  /// @brief This just gets read in from a yaml file
+  /// @brief Is the sample for when operating in Reverse Horn Current, read in from sample config
```

Comment:
pet peeve: what does RHC mean? better to refer to neutrino-mode and antineutrino-mode. Doesn't require people to remember which charge is focussed depending on the current flow through the horn conductors.

Reply:
agree in general with this. This variable has actually been removed as it's not needed anymore.